init_process.go

```diff
@@ -0,0 +1,406 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+// static file path
+const ConfigFileFolderPath = "init_process_fluent-bit_s3_config_files"
```

Comment:
Let's don't include both - and _. and make these starting with lower cases. I prefer:
```
const s3FilePath = "fluent-bit-init-s3-files"
const mainConfigFilePath = "fluent-bit-init.conf"
const originalMainConfigPath = "/fluent-bit/etc/fluent-bit.conf"
const invokerFilePath = "invoker.sh"
```

Reply:
Already modified